Docs/source/building/building.rst

```diff
@@ -53,6 +53,7 @@ options are:
     * ``USE_OMP=TRUE`` or ``FALSE``: Whether to compile with OpenMP support.
     * ``USE_GPU=TRUE`` or ``FALSE``: Whether to compile for Nvidia GPUs (requires CUDA).
     * ``USE_OPENPMD=TRUE`` or ``FALSE``: Whether to support openPMD for I/O (requires openPMD-api).
+    * ``MPI_THREAD_MULTIPLE=TRUE`` or ``FALSE``: Whether to initialize MPI with thread multiple support. Required to use asynchronous IO with more than 64 MPI tasks.
```

Comment:
I think it's formally from looking that the AMReX source for IO with more than 64 "IO" tasks (`amrex.async_out_nfiles`), maybe @kngott can clarify?

Reply:
`MPI_THREAD_MULTIPLE=TRUE` is required when running Async I/O with more than `amrex.async_out_nfiles` tasks/files, or 64 ranks if `amrex.async_out_nfiles` is not specified.

So, for example, if you set `amrex.async_out_nfiles=16`, and were running with `amrex.async_out=1`, you would need to have compiled with `MPI_THREAD_MULTIPLE=TRUE` for cases with 17 or more ranks, instead of 65 or more.

It's been tricky to describe that succinctly, so feel free to give it your best shot. :)